.circleci/config.yml

```diff
@@ -274,6 +308,9 @@ workflows:
       - build_main_and_push_to_ecr:
           requires:
             - build_and_test
+      - scan_image:
+          requires:
+            - build_main_and_push_to_ecr
       - deploy_main_to_dev:
           requires:
             - build_main_and_push_to_ecr
```

Comment:
You might want to make this dependent on `scan_image` running, the same way it's done in CT.